qstash/api/events/list.mdx

```diff
@@ -65,11 +65,47 @@ The name of the endpoint if this message was sent through a topic
 </ResponseField>
 
 <RequestExample>
-```sh
+```sh curl
 curl https://qstash.upstash.io/v2/events \
   -H "Authorization: Bearer <token>"
 ```
 
+```javascript Node
+fetch('https://qstash.upstash.io/v2/events', {
+  headers: {
+    'Authorization': 'Bearer <token>'
+  }
+});
+```
+
+```python Python 
+import requests
+headers = {
+    'Authorization': 'Bearer <token>',
+}
+
+response = requests.get('https://qstash.upstash.io/v2/events', headers=headers)
+```
+
+```go Go
+client := &http.Client{}
+req, err := http.NewRequest("GET", "https://qstash.upstash.io/v2/events", nil)
+if err != nil {
+  log.Fatal(err)
+}
+req.Header.Set("Authorization", "Bearer <token>")
+resp, err := client.Do(req)
+if err != nil {
+  log.Fatal(err)
+}
+defer resp.Body.Close()
+bodyText, err := io.ReadAll(resp.Body)
+if err != nil {
+  log.Fatal(err)
+}
+fmt.Printf("%s\n", bodyText) 
+```
```

Comment:
To make the examples consistent with each other, I think we can do this. Same applies to other go snippets
```suggestion
```go Go
req, err := http.NewRequest("GET", "https://qstash.upstash.io/v2/events", nil)
if err != nil {
  log.Fatal(err)
}
req.Header.Set("Authorization", "Bearer <token>")
resp, err := http.DefaultClient.Do(req)
if err != nil {
  log.Fatal(err)
}
defer resp.Body.Close()
```
```